R/applications.R

```diff
@@ -43,7 +43,7 @@ applications <- function(account = NULL, server = NULL) {
   serverDetails <- serverInfo(accountDetails$server)
   client <- clientForAccount(accountDetails)
 
-  isConnect <- !isShinyapps(accountDetails)
+  isConnect <- !isConnectApp(accountDetails$server)
```

Comment:
Even if this is as intended it's confusing to have a variable named `isConnect` which actually tells us the opposite (if it's *not* Connect). Rename?

Reply:
good catch! i actually missed that this does something completely different after the refactor, this is just wrong.